### PR TITLE
Add augmentors to chip classification

### PR DESCRIPTION
**Pull request by lewfish**
This PR adds augmentation options for PyTorch chip classification using the albumentations library.

I tested this by using the `ToGray` augmentation on a sample chip classification experiment and saw that some of the debug chips were greyscale.

This supercedes #851 

Connects #831

### CODE REVIEW

**Comment by lewfish** [comment]
@lmbak I made some changes to get this over the line faster. The main change is that I created an `AlbumentationsDataset` adapter which makes it so we don't need to make any changes to the original `Dataset`.

**Comment by lewfish** [comment]
Also, I made it so the validation dataset doesn't have any data augmentations applied to it. This is a best practice that reduces the variance in the validation metrics from one run to the next.

**Comment by lmbak** [comment]
Great! Thanks for the feedback. In line with your comment I'm assuming that oversampling is also not applied to the validation set, I'll fix that in the corresponding PR.
